File: class09/cpp/AP0901-TrocaValores.cpp
#include <iostream>
int main() {
    int a = 5, b = 8, temp;
    std::cout << "Antes: a=" << a << ", b=" << b << std::endl;
    temp = a; a = b; b = temp;
    std::cout << "Depois: a=" << a << ", b=" << b << std::endl;
    return 0;
}

File: class09/java/AP0901-TrocaValores.java
public class Troca {
    public static void main(String[] args) {
        int a = 5, b = 8, temp;
        System.out.println("Antes: a=" + a + ", b=" + b);
        temp = a; a = b; b = temp;
        System.out.println("Depois: a=" + a + ", b=" + b);
    }
}
